Packs/JoeSecurity/Integrations/JoeSecurityV2/JoeSecurityV2.yml

```diff
@@ -0,0 +1,366 @@
+      type: Boolean
+    - contextPath: Joe.Analysis.runs.system
+      description: 'The run operation system.'
+      type: String
+    - contextPath: Joe.Analysis.runs.yara
+      description: 'The run yara.'
+      type: Boolean
+    - contextPath: Joe.Analysis.score
+      description: 'The run score.'
+      type: Number
+    - contextPath: Joe.Analysis.scriptname
+      description: 'The run script name.'
+      type: String
+    - contextPath: Joe.Analysis.sha1
+      description: 'The file sha1.'
+      type: String
+    - contextPath: Joe.Analysis.sha256
+      description: 'The file sha256.'
+      type: String
+    - contextPath: Joe.Analysis.status
+      description: 'The status is one of submitted, running, finished.'
+      type: String
+    - contextPath: Joe.Analysis.threatname
+      description: 'The analysis threat name.'
+      type: String
+    - contextPath: Joe.Analysis.time
+      description: 'The analysis time.'
+      type: Date
+    - contextPath: Joe.Analysis.webid
+      description: 'The web id from joe security.'
+      type: String
+  - name: file
+    arguments:
+      - name: file
+        description: 'A comma-separated list of file names, SHA1, SHA256 or MD5 hashes.'
+        isArray: false
+        required: true
+        secret: false
+    deprecated: false
+    description: Retrieves files information from Joe Security.
+    execution: false
+    outputs:
+      - contextPath: File.Name
+        description: Name of the file.
+        type: String
+      - contextPath: File.MD5
+        description: MD5 hash of the file.
+        type: String
+      - contextPath: File.SHA1
+        description: SHA1 hash of the file.
+        type: String
+      - contextPath: File.SHA256
+        description: SHA256 hash of the file.
+        type: String
+      - contextPath: File.Tags
+        description: Tags of the file.
+        type: String
+      - contextPath: File.Relationships.EntityA
+        description: The source of the relationship.
+        type: String
+      - contextPath: File.Relationships.EntityB
+        description: The destination of the relationship.
+        type: String
+      - contextPath: File.Relationships.Relationship
+        description: The name of the relationship.
+        type: String
+      - contextPath: File.Relationships.EntityAType
+        description: The type of the source of the relationship.
+        type: String
+      - contextPath: File.Relationships.EntityBType
+        description: The type of the destination of the relationship.
+        type: String
+  - name: url
+    arguments:
+      - name: url
+        description: 'A comma-separated list of urls.'
+        isArray: false
+        required: true
+        secret: false
+    deprecated: false
+    description: Retrieves urls information from Joe Security.
+    execution: false
+    outputs:
+      - contextPath: URL.Name
+        description: 'Name of the url.'
+        type: String
```

Comment:
what other outputs does this command have?
and we want relationships outputs as well?

Reply:
I will add the relationships but the only output is the url itself and the Dbotscore.